src/app/camo-card/camo-card.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="card">
+    <div class="progress-number">
+      {{ progressData[weapon.id].current }} /
```

Comment:
now use the `progress` @Input() here instead of progressData. Pass the progressData[weapon.id] from the home-page.component.html